pkg/database/token.go

```diff
@@ -204,6 +204,19 @@ func (db *Database) VerifyCodeAndIssueToken(realmID uint, verCode string, accept
 			return err
 		}
 
+		// Log the issued code.
+		now := time.Now().Truncate(24 * time.Hour)
```

Comment:
I think we might want to do this inside a callback instead of here.

Reply:
agreed - I think it's best to not have stats update required in the transaction

Reply:
Done